scripts/lightning.json

```diff
@@ -1117,36 +1136,118 @@
               "refId": "A"
             },
             {
-              "expr": "rate(lightning_block_read_seconds_sum[30s]) / rate(lightning_block_read_seconds_count[30s])",
+              "expr": "rate(lightning_block_deliver_bytes_sum[30s])",
               "format": "time_series",
               "intervalFactor": 2,
-              "legendFormat": "block read",
+              "legendFormat": "block deliver",
               "refId": "B"
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeShift": null,
+          "title": "SQL process rate",
+          "tooltip": {
+            "shared": true,
+            "sort": 0,
+            "value_type": "individual"
+          },
+          "type": "graph",
+          "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "Bps",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
             },
             {
-              "expr": "rate(lightning_block_deliver_seconds_sum[30s]) / rate(lightning_block_deliver_seconds_count[30s])",
+              "format": "short",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            }
+          ]
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_LIGHTNING}",
+          "fill": 1,
+          "id": 17,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "span": 6,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "lightning_block_read_bytes_sum",
               "format": "time_series",
               "intervalFactor": 2,
-              "legendFormat": "block deliver",
+              "legendFormat": "read",
+              "refId": "A"
+            },
+            {
+              "expr": "lightning_block_deliver_bytes_sum",
+              "format": "time_series",
+              "intervalFactor": 2,
+              "legendFormat": "deliver",
+              "refId": "B"
+            },
+            {
+              "expr": "pd_cluster_status{type=\"storage_size\"} / 3",
```

Comment:
Perhaps hide this until we can get the number of replicas from Prometheus (to avoid hard-coding the "3").